doc/api/stream.md

```diff
@@ -1061,7 +1061,7 @@ Examples of Transform streams include:
 <!--type=misc-->
 
 The `stream` module API has been designed to make it possible to easily
-implement streams using JavaScript's prototypical inheritance model.
+implement streams using JavaScript's prototypal inheritance model.
```

Comment:
http://www.merriam-webster.com/dictionary/prototypal ;)

(not complaining, I just giggled a bit at this. :smile:)